service-app/internal/api/handler.go

```diff
@@ -294,6 +294,9 @@ func (c *IndexController) IngestHandler(w http.ResponseWriter, r *http.Request)
 	// Wait for the internal job queue to finish processing.
 	c.Queue.Wait()
 
+	// Clear errors
+	c.Queue.ClearErrors()
+
 	// Check if any errors were collected from the internal jobs.
 	jobErrors := c.Queue.GetErrors()
```

Comment:
Should these not be the over way around? Otherwise we're always going to find `q.errors` empty?

Reply:
Whoops, got ahead of myself !